products/contour.md

```diff
@@ -20,39 +20,45 @@ identifiers:
 
 # eol(X) = releaseDate(X+3)
 releases:
+-   releaseCycle: "1.30"
+    releaseDate: 2024-08-01
+    eol: false # releaseDate(1.33)
+    latest: "1.30.0"
+    latestReleaseDate: 2024-08-01
+
 -   releaseCycle: "1.29"
     releaseDate: 2024-05-07
-    eol: false
+    eol: false # releaseDate(1.32)
     latest: "1.29.2"
     latestReleaseDate: 2024-07-31
 
 -   releaseCycle: "1.28"
     releaseDate: 2024-02-12
-    eol: false
+    eol: false # releaseDate(1.31)
     latest: "1.28.6"
     latestReleaseDate: 2024-07-31
 
 -   releaseCycle: "1.27"
     releaseDate: 2023-10-30
-    eol: false
+    eol: 2024-08-01 # releaseDate(1.30)
```

Comment:
```suggestion
    eol: 2024-07-31 # releaseDate(1.30)
```

According to https://github.com/projectcontour/contour/releases/tag/v1.30.0 (normalize to GMT, the timezone used by our automation).